Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/human.py

```diff
@@ -0,0 +1,42 @@
+class Human:
+    def __init__(self, name: str, age: int, gender: str):
+        self.__name = name
+        self.__age = age
+        self.__gender = gender
+        self.__status = "alive"
+        self.__age_limit = 100
+
+    def grow(self):
+        self.__is_alive()
+        if self.__age < self.__age_limit:
+            self.__age += 1
+        else:
+            self.__status = "dead"
+
+    def __is_alive(self):
+        if self.__status == "alive":
+            return True
+        else:
+            raise Exception(f"{self.__name} is already dead...")
+
+    def change_gender(self, gender: str):
+        self.__is_alive()
```

Comment:
no test for this step. Try change gender for dead person

Reply:
Added test with dead person